srcpkgs/kget/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'kget'
+pkgname=kget
+version=18.04.3
+revision=1
+build_style=cmake
+configure_args="-DDESKTOPTOJSON_EXECUTABLE=/usr/bin/desktoptojson"
+hostmakedepends="extra-cmake-modules python qt5-qmake qt5-host-tools
```

Comment:
All but extra-cmake-modules, pkg-config could be conditional, like in https://github.com/void-linux/void-packages/blob/master/srcpkgs/ark/template
Same for DESKTOPTOJSON_EXECUTABLE.

Reply:
It thought the current consensus was to avoid unnecessary conditionals?

Reply:
@maxice8 @Gottox Which way is preferred?

Reply:
@Chocimier will depend on who you ask, i prefer conditionals